dashboard/app/models/pd/scholarship_info.rb

```diff
@@ -43,4 +43,8 @@ def self.update_or_create(user, application_year, course, scholarship_status)
       Pd::ScholarshipInfo.new(user: user, application_year: application_year, course: course)
     scholarship_info.update(scholarship_status: scholarship_status)
   end
+
+  def friendly_status
```

Comment:
Can this function name explain a bit more about what it means?  In isolation, `friendly_status` could mean a lot of things! :)

Reply:
how about `display_status` or `status_friendly_name`?

Reply:
I'm not sure.  I guess there is some precedent for putting `friendly_` in front of a regular function, so maybe the existing is okay.  Or how about `get_friendly_status`?  Or maybe just adding a comment explaining what the function does? :)